src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class AbstractFigure implements Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```

create interface Drawable with method void `draw();`